scripts/config_whitelist.txt

```diff
@@ -556,7 +556,6 @@ CONFIG_FORCE_DDR_DATA_BUS_WIDTH_32
 CONFIG_FORMIKE
 CONFIG_FPGA_COUNT
 CONFIG_FPGA_DELAY
-CONFIG_FPGA_LATTICE
```

Comment:
I'm not sure I understand this change, why remove this from the whitelist?

Reply:
I don't know !-) lol When removing the FPGA programming way back when, it must have seemed it was needed for some reason. I put it back.

Reply:
I am curious:  What is CONFIG_FORMIKE supposed to do?

Reply:
@luckshot : In `drivers/video`, it includes a module called `drivers/video/formike.c`. See commit `b26354`.